src/makeCalculator.js

```diff
@@ -1,10 +1,41 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+      return this;
+    },
+
+    subtract(value) {
+      this.result -= value;
+      return this;
+    },
+
+    multiply(value) {
+      this.result *= value;
+      return this;
+    },
+
+    divide(value) {
+      this.result /= value;
```

Comment:
You need to add a check for 0, because you cannot divide by 0 according to the rules of mathematics